extension-testcontainer/src/main/java/io.camunda.zeebe.process.test.extension.testcontainer/ContainerizedEngine.java

```diff
@@ -58,6 +72,10 @@ public void stop() {
     closeChannel(channel);
   }
 
+  /**
+   * Reset the test engine. Implementation wise a reset will result in stopping the old engine and
+   * creating a new one.
+   */
```

Comment:
🔧 Personal opinion:

>Reset the test engine. A reset stops the current engine, and replaces it with a new engine. Note that you'll need to redeploy your processes because it is an entirely new engine.